src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -60,6 +61,8 @@ public string ClientVersion
 
         public WindowsBrokerOptions WindowsBrokerOptions { get; set; }

```

Comment:
We should not keep these around any longer. Internally we should only keep `BrokerOptions `

Reply:
These have been renamed to UwpBrokerOptions. They are kept for backward compatibility.